model/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+load("@rules_proto//proto:defs.bzl", "proto_library")
```

Comment:
Is it conventional to name this file ``BUILD.bazel`` or ``BUILD``? I prefer the latter, but that might just be because it's in-line with conventions I'm used to. We should go with whatever the Bazel project recommends.

Reply:
It is convention to name the build files `BUILD` but at the moment naming files with this convention conflicts with the naming of some of Gradle's generated files. I believe that we need to name our build files for Bazel 'BUILD.bazel' while we are still using Gradle. In retrospect, I should have added this to the explanation portion of the PR (and can still add it if you think that it is important).

Reply:
Done! See the PR description for a link to this issue :) I will also update the other branches and add the necessary TODOs.